src/trace_writer.ml

```diff
@@ -655,6 +636,109 @@ let check_current_symbol
     Callstack.push thread_info.callstack location.symbol
 ;;
 
+module Ocaml_hacks : sig
```

Comment:
Could you leave a comment on this module describing your hacks? Maybe include a link to the relevant ocaml/ocaml PR that added the elf notes somewhere.

Reply:
Sure, done.